### PR TITLE
Added link to the RSS feed of the blog

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: antora-ui-camel/public/_/partials/footer-content.hbs [hunk 1]
```diff
@@ -11,6 +11,7 @@
             <dl>
                 <dt><label for="footer-toggle-overview">Overview</label><label for="footer-toggle-overview">&#65291;</label></dt>
                 <dd><a href="{{siteRootPath}}/blog/">Blog</a></dd>
+                <dd><a href="{{siteRootPath}}/blog/index.xml">RSS Feed (Blog)</a></dd>
```
Comment (zregvart) [inline antora-ui-camel/public/_/partials/footer-content.hbs:14]: Why not do something like:
```suggestion
                <dd><a href="{{siteRootPath}}/blog/">Blog</a> (<a href="{{siteRootPath}}/blog/index.xml">RSS Feed</a>)</dd>
```

Comment (orpiske) [reply antora-ui-camel/public/_/partials/footer-content.hbs:14]: Sounds good to me, thanks!

**Review by davsclaus — APPROVED** [review]

**Comment by orpiske** [comment]
It looks like it didn't work. I'll take a look later.

**Comment by zregvart** [comment]
This change must be made in https://github.com/apache/camel-website/blob/main/antora-ui-camel/src/partials/footer-content.hbs and then the UI needs to be rebuilt (`yarn workspace antora-ui-camel build`), also a corresponding change must be made to the [Hugo template](https://github.com/apache/camel-website/blob/main/antora-playbook-snippets/antora-playbook.yml).

**Comment by orpiske** [comment]
> This change must be made in https://github.com/apache/camel-website/blob/main/antora-ui-camel/src/partials/footer-content.hbs and then the UI needs to be rebuilt (`yarn workspace antora-ui-camel build`), also a corresponding change must be made to the [Hugo template](https://github.com/apache/camel-website/blob/main/antora-playbook-snippets/antora-playbook.yml).

Thanks for the heads up. Let's close this one for now as I don't have these web stuff installed here.

**Pull request closed without merging**